Plugin refactoring

Yuri2b opened the pull request:
- Extracting images and wrapping then to Gatsby nodes
- Introducing new node types

peterliapin commented on constants.js:
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  // GraphQL nodes type for entities with type "post"
+  POST_NODE_TYPE: "OnlinesalesPost",
```
Onlinesales -> OnlineSales - this is a proper way to write the name of the product

Yuri2b replied:
Fixed

peterliapin commented on gatsby-node.js:
```diff
@@ -1,95 +1,129 @@
+const getNodeType = (type) =>
+  type === "post" ? POST_NODE_TYPE : PAGE_NODE_TYPE;
+
+const getNodeLanguage = (language) =>
```
@Yuri2b it is OK to keep it this way for now, but moving forward we should consider there could be other languages than ru and en, what happens than?

Yuri2b replied:
I've added simple parsing for 2 chars language code, assuming that CMS provides ISO 639-1 or BCP 47 languge code format (e.g. "en" or "en-US")

Yuri2b committed:
Plugin refactoring

- Extracting images and wrapping then to Gatsby nodes
- Introducing new node types

peterliapin committed:
Update README.md

peterliapin committed:
Update README.md
